src/compas_ui/scene.py

```diff
@@ -18,24 +19,26 @@ def clear_scene(self):
     raise NotImplementedError
 
 
-class Scene(object):
+class Scene(Data, Singleton):
```

Comment:
i think we should make a parent class for `Data`, which is called `Serializable`, and use this for `Scene`, `Object` and so on...